test/integration/narrow/routes/irrigation-water-source.test.js

```diff
@@ -1,38 +1,60 @@
-const { getCookieHeader, getCrumbCookie } = require('./test-helper')
+const { crumbToken } = require('./test-helper')
+
 describe('Irrigation water source page', () => {
-  let crumCookie
+  const project = ['some fake project']
+  const irrigatedCrops = 'some fake crop'
+  const irrigatedLandCurrent = '123'
+  const irrigatedLandTarget = '456'
+  const waterSourceCurrent = ['some source 1']
+  const waterSourcePlanned = ['some source 2', 'another source']
+
+  const session = require('../../../../app/helpers/session')
+
+  jest.mock('../../../../app/helpers/session', () => ({
+    setYarValue: (request, key, value) => null,
+    getYarValue: (request, key) => {
+      switch (key) {
```

Comment:
repeatative code. Can be moved to function which takes dictionary n return value.

Reply:
its a mock function its using different values on different pages and we might need to change it frequently. thats why i didnt put it in function

Reply:
Yes. thats why I said dictionary, so that you can set dictionary value in each test. But function is common to all.

Reply:
As discussed, please add tech debt.